pkg/dogstatsd/listeners/udp.go

```diff
@@ -96,7 +78,6 @@ func NewUDPListener(packetOut chan Packets, sharedPacketPool *PacketPool) (*UDPL
 func (l *UDPListener) Listen() {
 	log.Infof("dogstatsd-udp: starting to listen on %s", l.conn.LocalAddr())
 	for {
-		udpPackets.Add(1)
```

Comment:
The behavior is not exactly the same as before but IMO counting udp packets should be done after `ReadFrom`.